lib/opentok/opentok.rb

```diff
@@ -177,6 +177,10 @@ def sip
       @sip ||= Sip.new client
     end
 
+    def connection
+      @connection ||= Connections.new client
```

Comment:
Getting error in this line: "NameError - uninitialized constant OpenTok::OpenTok::Connections", I think the require statement is missing `require "opentok/connections"`